app/assets/src/components/views/phylo_tree/PhyloTreeViz.jsx

```diff
@@ -25,7 +24,6 @@ class PhyloTreeViz extends React.Component {
     if (this.phyloTree && this.phyloTree.newick) {
       let newick = this.phyloTree.newick;
       // below from http://bl.ocks.org/spond/f6b51aa6f34561f7006f
-      let _ = require("lodash");
```

Comment:
You probably still need this actually?

Reply:
why? doesn't it just load all of lodash module into the _ variable? lmk if i'm missing something.

Reply:
Ah nvm. I thought it was needed to import the library